bench.sh

```diff
@@ -23,7 +23,7 @@ alloc_libs="sys="      # mapping from allocator to its .so as "<allocator>=<sofi
 readonly tests_all1="cfrac espresso barnes redis lean larson-sized mstress rptest gs"
 readonly tests_all2="alloc-test sh6bench sh8bench xmalloc-test cscratch glibc-simple glibc-thread"
 readonly tests_all3="larson lean-mathlib malloc-large mleak rbstress cthrash"
-readonly tests_all4="z3 spec spec-bench"
```

Comment:
Could you add it to `tests_all2` so we get some runs in CI?  At the moment, CI is only testing building it, and not running.  We might not want it in all CI configs if it takes too long, but I would like to see it before we merge.

Reply:
Done